staging/src/k8s.io/client-go/tools/remotecommand/v4_test.go

```diff
@@ -64,7 +65,9 @@ func TestV4ErrorDecoder(t *testing.T) {
 		if want == "" {
 			want = "<nil>"
 		}
-		if got := fmt.Sprintf("%v", err); got != want {
+		got := fmt.Sprintf("%v", err)
```

Comment:
checking `strings.Contains(got, want)` and changing the expected error to just `unexpected end of JSON input` is probably sufficient